Assets/Scripts/Selling/SellingPrompt.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Written by Ellie McDonald
+// 04/22/21 
+// This class will control when the selling canvas are enabled 
+using UnityEngine;
+
+public class SellingPrompt : MonoBehaviour
+{
+    private Vector3 player, sellingShed;
+    private Canvas sellPotionCanvas, potionInventoryCanvas;
+
+    private bool enablePrompt = false;
+    private AbstractCalculate distance = new ThreeDimensionalCalculateSellingShed();
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        player = GameObject.Find("Male A").transform.position;
+        sellingShed = GameObject.Find("Selling Shed").transform.position;
+        sellPotionCanvas = GameObject.Find("SellPotionCanvas").GetComponent<Canvas>();
+        potionInventoryCanvas = GameObject.Find("PotionInventoryCanvas").GetComponent<Canvas>();
+    }
+
+    void OnGUI()
+    {
+        player = GameObject.Find("Male A").transform.position;
+
+        // this technically shouldn't change positions so idk if this needs to be int the OnGui() or not
+        sellingShed = GameObject.Find("Selling Shed").transform.position;
```

Comment:
I would remove from ongui since ongui runs every frame similar to update. If it only needs to be done once, put it in start.